fe/fe-core/src/main/java/com/starrocks/catalog/JDBCTable.java

```diff
@@ -278,7 +278,7 @@ public ProtocolType getProtocolType() {
     public enum ProtocolType {
         UNKNOWN,
         MYSQL,
-        POSTGRES,
+        POSTGRESQL,
```

Comment:
I'm not againist this change, but just being curious, what's the nuance between postgres and postresql?

Reply:
Hi @dirtysalt good question, I thought this would be a bugfix. 

ProtocolType is used in `getProtocolType` function when there is an input jdbc url: https://github.com/StarRocks/starrocks/blob/main/fe/fe-core/src/main/java/com/starrocks/catalog/JDBCTable.java#L259

e.g. if JDBC url is `jdbc:mysql:host` -> return `ProtocolType.MYSQL` 

For Postgres, the jdbc url format seems like it should be `jdbc:postgresql:host`: https://jdbc.postgresql.org/documentation/use/#connecting-to-the-database

So before, if ProtocolType is `POSTGRES` this would return `UNKNOWN` for Postgres JDBC urls, and before it expects `jdbc:postgres:host` 🤔 

What I can do instead is add another `POSTGRESQL` field instead of deleting the old `POSTGRES` enum for safety.